app/views/gradebooks/view.html.erb

```diff
@@ -58,9 +58,18 @@
   </style>
 <% end %>
 
-<% content_for :javascripts do %>
+
+<% content_for :js_widget_patch do %>
   <%= external_javascript_include_tag "jquery-ui", "1.10.2" %>
-  <%= external_javascript_include_tag "bootstrap-tooltip", "2.3.1" %>
+  <%# jQuery UI is required for SlickGrid, but Bootstrap provides a tooltip plugin %>
+  <%# The following must come after jQuery UI but before Bootstrap 3 is loaded %>
+  <%# to resolve namespace issues %>
+  <script>
+    $.widget.bridge('uitooltip', $.ui.tooltip);
```

Comment:
can't we use bootstrap tooltips instead of jquery-ui? any idea what else is jquery-ui doing here?